skill_selectors/rule_based_selector/connector.py

```diff
@@ -126,9 +127,8 @@ async def send(self, payload: Dict, callback: Callable):
                 skills_for_uttr.append("dummy_skill")
                 # process intent with corresponding IntentResponder
                 skills_for_uttr.append("dff_intent_responder_skill")
-            elif command_detected:
+            if embodied_cmd_detected or command_detected:
```

Comment:
elif will work if you delete your intents from high_priority_intents where you added (i wrote above that it is not neccessary)

Reply:
elif readded